pkg/api/util/doget.go

```diff
@@ -93,13 +93,13 @@ func DoPostChunked(c *http.Client, url string, contentType string, body io.Reade
 	defer r.Body.Close()
 
 	var m int
+	buf := make([]byte, 128)
```

Comment:
Nit. Any reason we use such a low size here?

Reply:
I don't have a good reason. I just assumed it would be ok to let it resize itself (especially since since the initiator is the CLI - the overhead has no meaningful impact). Is there a bigger size you would prefer to put in here?

Reply:
Got it, yes, in that case it's not particularly meaningful. Personally I wouldn't question a 4k buffer, but for CLI likely not very important. Noticed the comment to `DoPostChunked` mentions `GET`, mind updating that?

Reply:
ah yes - good catch. I had changed it from a `GET` to a `POST` sometime in development and missed that comment. 
While i'm here I'll make the buffer anyway. (4k sounds good)